src/compiler/compile/utils/is_interactive_element.ts

```diff
@@ -0,0 +1,48 @@
+import Element from '../nodes/Element';
+
+const interactiveInputTypes = new Set([
+	'submit',
+	'reset',
+	'image',
+	'button',
+	'radio',
+	'checkbox'
+]);
+
+export const is_interactive_element: (element: Element) => boolean = function (
+	element
+) {
+	if (element.name === 'details') {
```

Comment:
Just a suggestion: I think you can use a `Set` for the tag names which return true to reduce code surface, like you did with `interactiveInputTypes`.

```ts
const interactiveElements = new Set([
    'details',
    'summary',
    // ...
]);

export const is_interactive_element: (element: Element) => boolean = function (
	element
) {
    if (interactiveElements.has(element.name)) {
        return true;
    }

    // ...
}
```

Reply:
Thank you for reviewing! When I made the `is_interactive_element` module, I referred to [interactive elements](https://html.spec.whatwg.org/multipage/interactive-elements.html). I didn't use `Set` because I wanted to implement it as it is in the [interactive elements](https://html.spec.whatwg.org/multipage/interactive-elements.html) so that it can be easily reflected when there is a change in the[interactive elements](https://html.spec.whatwg.org/multipage/interactive-elements.html), but I think that it is also good to implement it using `Set` separately. How about do you think?

Reply:
Oh interesting. I was just skimming through and noted this, an actual contributor can probably chime in on this. ;)